pkg/rancher-desktop/backend/k3sHelper.ts

```diff
@@ -975,6 +975,7 @@ export default class K3sHelper extends events.EventEmitter {
       merge(userConfig.users, workConfig.users);
       merge(userConfig.clusters, workConfig.clusters);
       userConfig.currentContext ??= contextName;
+      userConfig.currentContext = userConfig.currentContext === "" ? contextName : userConfig.currentContext;
```

Comment:
```suggestion
      userConfig.currentContext ||= contextName;
```
In JS, empty-string is falsy, so we can just shortcut it.